projects/nae-example-app/src/app/doc/public-task-view/public-task-view.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="task-tab-background full-height">
     <div fxLayout="column" fxLayoutAlign="start stretch" class="content-margin full-height" >
-        <nc-header #header type="task" class="task-panel-padding-mini"></nc-header>
+        <nc-header #header [hidden]="!!config && config['header'] === 'false'" type="task" class="task-panel-padding-mini"></nc-header>
```

Comment:
the `AbstractPanelComponent` defines a method `showPanelHeader()` that seems to be doing a similar thing. Why is this method not called here?

Reply:
Corrected.